activerecord/test/cases/touch_later_test.rb

```diff
@@ -135,5 +138,7 @@ def test_touching_through_nested_attributes
     owner.update pets_attributes: { "0" => { id: "1", name: "Alfred" } }
 
     assert_not_equal time.to_i, owner.reload.updated_at.to_i
+  ensure
+    ActiveRecord.before_committed_on_all_records = original
```

Comment:
Can we duplicate that test with the flag set to `false`?

Reply:
Done!